module/Olcs/src/Controller/IndexController.php

```diff
@@ -263,6 +263,11 @@ public function entityListAction()
                 ->setIrhpPermitStock($value);
                 $results = ['' => 'Please select'] + $srv->fetchListOptions();
                 break;
+            case '':
```

Comment:
Can we review what type will be here

Reply:
The '' case isn't needed, just left over from a previous experimentation, have checked and cleaned up now.

There may be a case for making the Licence / Application query behaviour more similar to the other cases populated by queries (so the code in the controller's alterFormForAdd() would be refactored to align with the getListDataUser() and getListDataDocTemplates())

As a Random Act of Kindness we could remove the repeated fetchListOptions() by replacing with a single array and addition (where needed) before the format & return